crates/python/qcs_sdk/__init__.pyi

```diff
@@ -11,6 +11,11 @@ from qcs_sdk.compiler.quilc import CompilerOpts
 
 from qcs_sdk.client import QCSClient as QCSClient
 
+from . import qpu as qpu
+from . import qvm as qvm
+from . import compiler as compiler
+from . import client as client
```

Comment:
This was the result of `stubtest` complaining about missing modules for these names after setting `__all__` in the root of `__init__.py`. It turns out the way we defined them were only valid as packages and re-exporting them here makes them proper sub-modules. That makes sense, but `stubtest` never complained about it before. Either way, happy that we stumbled into this, as I think it'll resolve some inconsistent behavior I've seen with different tools and import styles.